examples/assets_pandas_pyspark/assets_pandas_pyspark/local_spark_filesystem_io_manager.py

```diff
@@ -70,7 +70,7 @@ def load_input(self, context) -> Union[PandasDF, SparkDF]:
             return pd.concat(map(pd.read_csv, paths))
         elif context.dagster_type.typing_type == SparkDF:
             return (
-                SparkSession.builder.getOrCreate()
+                SparkSession.builder.getOrCreate()  # type: ignore
```

Comment:
pyright doesn't like something about the new classproperty here in the latest version: https://github.com/apache/spark/blob/master/python/pyspark/sql/session.py#L495-L496

I would be lying if I said I understood why some SparkSession.builder calls hit the type issue and others do not though

Reply:
>I would be lying if I said I understood why some SparkSession.builder calls hit the type issue and others do not though

Would need to see an example of one that doesn't hit but my guess is that it's inside an unannotated function.